pkg/apis/apirator/v1alpha1/apimock_types.go

```diff
@@ -25,9 +25,14 @@ import (
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
 const (
-	PROVISIONED = "PROVISIONED"
-	ERROR       = "ERROR"
-	INVALID_OAS = "INVALID_OAS"
+	PROVISIONED         = "PROVISIONED"
+	ERROR               = "ERROR"
+	InvalidOas          = "INVALID_OAS"
```

Comment:
Words in names that are acronyms should have a consistent case. For example, `OAS` should appear as `OAS` or `oas` (as in `oasError`, or `OASError`), never as `Oas`.